src/libraries/System.Text.RegularExpressions/tests/MonoRegexTests.cs

```diff
@@ -1030,7 +1015,6 @@ public static IEnumerable<object[]> RegexTestCases()
             yield return new object[] { @"^(((?<foo>\()[^()]*)+((?<bar-foo>\))[^()]*)+)+(?(foo)(?!))$", RegexOptions.None, "((a(b))c))","Fail." };
             yield return new object[] { @"^(((?<foo>\()[^()]*)+((?<bar-foo>\))[^()]*)+)+(?(foo)(?!))$", RegexOptions.None, ")(","Fail." };
             yield return new object[] { @"^(((?<foo>\()[^()]*)+((?<bar-foo>\))[^()]*)+)+(?(foo)(?!))$", RegexOptions.None, "((a((b))c)","Fail." };
-            yield return new object[] { @"b", RegexOptions.RightToLeft, "babaaa", "Pass. Group[0]=(2,1)" };
```

Comment:
Where is this one duplicated?

Reply:
https://github.com/dotnet/runtime/blob/5e6711dbfb1b696e1fb7c53f9d84d33c60dc87be/src/libraries/System.Text.RegularExpressions/tests/MonoRegexTests.cs#L798